### ISSUE TITLE
RunningVar of Batch Norm comes from biased or unbiased estimator?

### ISSUE THREAD

**Issue by SuperbTUM** [pytorch/pytorch#106366]
### 🐛 Describe the bug

When I looked into the implementation of batch normalization, I found a minor inconsistency in running var. The doc says `The standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False).`, however, what comes out of `bn.running_var` is unexpectedly equivalent to `torch.var(input, unbiased=True)`, which is from unbiased estimator?

A quick demo is:
```python
x1 = torch.ones(4, 2, 3, 3)
x2 = torch.ones(4, 2, 3, 3) * 2
x3 = torch.ones(4, 2, 3, 3) * 3
x4 = torch.ones(4, 2, 3, 3) * 4
x = torch.cat((x1, x2, x3, x4), dim=0).cuda()
bn = nn.BatchNorm2d(2).cuda()
bn(x)
print(bn.running_var) # you will get 1.0259
batch_ch_var = torch.var(x, unbiased=True).mean(dim=(0,2,3))
running_var = running_var + self.momentum * (
                        batch_ch_var.data -running_var)# you will get 1.0259
batch_ch_var = torch.var(x, unbiased=False).mean(dim=(0,2,3))
running_var = running_var + self.momentum * (
                        batch_ch_var.data -running_var)# you will get 1.0250
```
where momentum=0.1 as default.

### Versions

`torch.__version__`
2.0.1+cu118

cc @albanD @mruberry @jbschlosser @walterddr @mikaylagawarecki

**Comment by mikaylagawarecki** [comment]
The docs have been [updated on main ](https://pytorch.org/docs/main/generated/torch.nn.BatchNorm2d.html#torch.nn.BatchNorm2d)to say 

> At train time in the forward pass, the standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False). However, the value stored in the moving average of the standard-deviation is calculated via the unbiased estimator, equivalent to torch.var(input, unbiased=True).

Which seems to be consistent with what you are seeing since you are accessing `running_var`. Let me know if this makes sense!

**Comment by SuperbTUM** [comment]
Thanks! That makes a lot of sense! I validated it with the updated description and looked good.
